daemon/cmld.c

```diff
@@ -1061,20 +1073,30 @@ cmld_start_a0(container_t *new_a0)
 	}
 
 	container_set_key(new_a0, A0_KEY);
-	if (container_start(new_a0))
+	if (container_start(new_a0)) {
+		audit_log_event(uuid_new("00000000-0000-0000-0000-000000000000"), FATAL, CMLD,
+				CONTAINER_MGMT, INTERNAL, FAIL, NULL, "c0 start failed");
```

Comment:
mem_leek uuid_new()! resulting pointer is never free'd. since the reference is not keept.
Why not just use container_get_uuid(new_a0) ?

Same for the next 4 audit_log_event calls...